NOTES-WINDOWS.md

```diff
@@ -99,31 +99,37 @@ check the INSTALL.md file.
 Installation directories
 ------------------------
 
-The default installation directories are derived from environment
-variables.
+On most Unix platform installation directories are determined at build time via
+constant defines.  On Windows platforms however, installation directories are
+determined via registry keys, as it is common practice to build OpenSSL and
+install it to a variety of locations.
```

Comment:
```suggestion
constant defines.  On Windows platforms however, it is also possible to
determine the installation directories via registry keys, as it is common
practice to build OpenSSL and install it to a variety of locations.
```

Reply:
I'm not sure about this, given that we no longer use the constant defines on windows (per previous conversation), I don't think the phrase "It is also" applies here.  With this change set, you either define OSSL_WINCTX and set the registry keys, set the OPENSSL_CONF, etc environment variables, or it just doesn't try to load anything.  I can add language about the environment variables here if you like

Reply:
The behaviour should NOT be changed from what we currently have if OSSL_WINCTX is not defined. That means these will be constant defines even on Windows. And AFAIK that is what the current code does.